pageserver/src/tenant/size.rs

```diff
@@ -183,6 +183,13 @@ pub(super) async fn gather_inputs(
             }
         }
 
+        // all timelines also have an end point
+        updates.push(Update {
+            lsn: last_record_lsn,
+            command: Command::EndOfBranch,
+            timeline_id: timeline.timeline_id,
+        });
```

Comment:
Causes a failure here: https://github.com/neondatabase/neon/blob/38a7f988701665c5f0ada5baff778d29764e6ab7/test_runner/regress/test_tenant_size.py#L192

Needs to be guarded against:
- on ancestor: `last_record_lsn IN interesting_lsns`
- on branch: `last_record_lsn == timeline.get_ancestor_lsn()`

I can PR a fix suggestion, since the test now has changes as well.

Reply:
PR merged in 93ef570